Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/PaalmanPingsMonteCarloAbsorption.py

```diff
@@ -294,10 +294,6 @@ def PyInit(self):
         self.setPropertyGroup('ContainerNumberDensityUnit', 'Container Material')
```

Comment:
This comment relates to the container material settings in general but this was the only line I could attach a comment to in GitHub...
I think if the user has selected shape=preset they may want to override the sample material but I think it's unlikely they will want to override the container material. So if it's not v difficult to achieve I think it would be best to grey these container material property fields out if "preset" is selected